packages/@aws-cdk/aws-config/test/managed-rules.test.ts

```diff
@@ -1,37 +1,57 @@
-import { expect, haveResource } from '@aws-cdk/assert-internal';
+import '@aws-cdk/assert-internal/jest';
 import * as sns from '@aws-cdk/aws-sns';
 import * as cdk from '@aws-cdk/core';
-import { Test } from 'nodeunit';
 import * as config from '../lib';
 
-export = {
-  'access keys rotated'(test: Test) {
+describe('access keys', () => {
+  test('rotated', () => {
     // GIVEN
     const stack = new cdk.Stack();
 
     // WHEN
     new config.AccessKeysRotated(stack, 'AccessKeys');
 
     // THEN
-    expect(stack).to(haveResource('AWS::Config::ConfigRule', {
+    expect(stack).toHaveResource('AWS::Config::ConfigRule', {
       Source: {
         Owner: 'AWS',
         SourceIdentifier: 'ACCESS_KEYS_ROTATED',
       },
-    }));
+    });
+  });
+
+  test('rotated with max age', () => {
```

Comment:
I added this test here to get past the 80% jest threshold for branches. I now see that I could have changed `jest.config.js` to accept a lower threshold instead. Wondering which solution I should keep at this point.

Reply:
This is fine for now, but going forward just lower the threshold. 
Unlike PRs where we try to improve the system, the goal of this PR is very narrow, i.e., move to jest. Otherwise, we can easily get rat holed and spend a ton of time here. 

Hope that makes sense.